client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -135,6 +135,69 @@ function mediaButton( editor ) {
 		return { isLoaded, onLoad };
 	} )();
 
+	function parseShortcode( content ) {
```

Comment:
Looking further, the selection won't be set, but it appears we're parsing via: `const parsed = MediaSerialization.deserialize( node );`